### PR TITLE
[IMP] Expenses: updating for new 16 design

### PR DESCRIPTION
**Pull request by larm-odoo**
Updating for new design and updating out of date instructions.

### CODE REVIEW

File: content/applications/finance/expenses.rst
Comment (samueljlieber) [inline content/applications/finance/expenses.rst:469]: Should _message_ be capitalized here? Is it on the UI?
```suggestion
   chatter. In an individual report, simply click :guilabel:`Send message` to open the message text
```

File: content/applications/finance/expenses.rst
Comment (samueljlieber) [inline content/applications/finance/expenses.rst:490]: Early line break
```suggestion
expense reports that have been approved and need to be posted, adjust the filters on the left side
so that only the :guilabel:`Approved` status is active (checked off).
```

File: content/applications/finance/expenses.rst
Comment (samueljlieber) [inline content/applications/finance/expenses.rst:579]: Early line break
```suggestion
First, specify the invoicing policy for each expense category. Go to :menuselection:`Expenses app
--> Configuration --> Expense Categories`. Click on the expense category to open the expense
```

File: content/applications/finance/expenses.rst
Comment (samueljlieber) [inline content/applications/finance/expenses.rst:608]: Early line breaks
```suggestion
   Selecting a :guilabel:`Customer to Reinvoice` when creating an expense is critical, since this is
   what causes automatic invoicing after an expense report is approved. The :guilabel:`Customer to
   Reinvoice` field can be modified *until an expense report is* **approved**, then the field is no
   longer able to be modified.
```

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @larm-odoo 👋 I have a number of technical changes throughout the doc, and some I am unsure of the change and need your insight!

Can you please make the changes per my suggestions below, push back up and then tag me for review again once the PR is ready? 

Thank you 🙂

File: content/applications/finance/expenses.rst
Comment (larm-odoo) [reply content/applications/finance/expenses.rst:33]: OK- for this and the edit above, since the form is a product form, and the name of the "thing" you are creating is a 'category' instead of a 'product' - there was room for confusion, which is why I did all the italics (because it's crazy to call a category a product, or have it function like one but call it the other).

File: content/applications/finance/expenses.rst
Comment (larm-odoo) [reply content/applications/finance/expenses.rst:153]: ok, I initially did because that's a weird thing to reference - you would expect the customer name to be the reference, not a sales order, but I understand- and agree, and changed it =)

File: content/applications/finance/expenses.rst
Comment (larm-odoo) [reply content/applications/finance/expenses.rst:346]: My logic was, that when you go to "my" reports, it is only "your "reports- but if you look at "all" reports (a different filter) and you are a manager, you see ALL reports- reports for other people you can approve, not just yours. So I thought it might be important to mention it was only "your" reports- even though I know that word does not belong in the doc .  But I took it out- hopefully people get it on their own, or from context =)

File: content/applications/finance/expenses.rst
Comment (larm-odoo) [reply content/applications/finance/expenses.rst:469]: It's the box that pops up that allows you to type in a message- but the box itself isn't labeled as anything, so I am using the term "message box" generically. I am not sure if there is a better word/description for it but that made the most sense to me. I left it lower case because that is not the proper name for it, and you don't see those words on the UI.

**Comment by larm-odoo** [comment]
All changes made, ready for you again, @samueljlieber!

File: content/applications/finance/expenses.rst
Comment (samueljlieber) [inline content/applications/finance/expenses.rst:59]: Line over 100 chars
```suggestion
   the :guilabel:`Re-Invoice Expenses` section. Otherwise, the :guilabel:`Sales Price` field is
   hidden.
```

**Review by samueljlieber — APPROVED** [review]
Hi @larm-odoo 👋 this PR is looking great! I just caught 4 small items, please fix these and then pass this onto final review 🙂 Approving now. Nice job!

Also on Line 7: remove extra space before "After".

**Comment by larm-odoo** [comment]
Thanks as always, @samueljlieber! Edits made, now sending your way, @StraubCreative !

File: content/applications/finance/expenses.rst
Comment (StraubCreative) [inline content/applications/finance/expenses.rst:515]:
```suggestion
.. important::
```

File: content/applications/finance/expenses.rst
Comment (StraubCreative) [inline content/applications/finance/expenses.rst:189]: this might get a gui bc it's the name of a dashboard, no?
```suggestion
First, in the main :guilabel:`Expenses` app dashboard view (this view can also be accessed from
```

File: content/applications/finance/expenses.rst
Comment (StraubCreative) [inline content/applications/finance/expenses.rst:190]: `My Expenses to Report` doesn't appear to be the name of the menu link...
```suggestion
:menuselection:`Expenses app --> My Expenses --> My Expenses`), click :guilabel:`Scan`,
```

File: content/applications/finance/expenses.rst
Comment (StraubCreative) [inline content/applications/finance/expenses.rst:191]: need a conjunction here
```suggestion
and a file explorer pops up. Navigate to the receipt to be uploaded, click on it, and then click
```

File: content/applications/finance/expenses.rst
Comment (StraubCreative) [inline content/applications/finance/expenses.rst:435]: The idiom "checked off" I don't think works in this doc. It's usually in reference to to-do lists and productivity, as in, something was "done," not so much "enabled." As well idioms have a tendency to translate poorly into other languages so I'd be be more specific throughout the doc on every instance it shows up, e.g.:
```suggestion
   :guilabel:`Submitted` filter enabled.
```

File: content/applications/finance/expenses.rst
Comment (StraubCreative) [inline content/applications/finance/expenses.rst:241]: wording
```suggestion
To add an internal reference on an expense category, click on the category to open the form.
```

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hi @larm-odoo 
These are good changes. Screenshots are great too.
Before we merge, can you do another pass and make sure we're not using idioms or shorthand in the instructions? I caught a couple spots (but not all) where wording could be improved or we were missing a conjunction or a noun. I know it's a long doc and very technical, however we want to make sure we're specific in our language throughout.
Thanks and tag me again when we're ready for another look 👍

File: content/applications/finance/expenses.rst
Comment (larm-odoo) [reply content/applications/finance/expenses.rst:435]: That makes a lot of sense- I changed all the check boxes where it applies to selecting filters/enabling items. I left the other "check" works when it was explaining how to select a few reports vs selecting the entire list (you'd click a different box for that). I *think* that makes sense- but if you'd like those parts changed as well, LMK.

Basically, "checked off" is gone, but "check the box" I kept.

File: content/applications/finance/expenses.rst
Comment (larm-odoo) [reply content/applications/finance/expenses.rst:190]: Oof- thanks for that catch- that was from the other versions, yikes! I thought I got them all- nope =)
Fixed now

**Review by StraubCreative — APPROVED** [review]
Looks good. Thanks @larm-odoo 👍 

@robodoo r+

**Pull request closed without merging**
